apps/console/src/features/server-configurations/settings/index.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content."
+ */
+
+export * from "./edit-connector";
```

Comment:
Let's avoid using index.ts file to export all. Directly import components from the relevant tsx file.

Reply:
Fixed for the settings file only. We will have to update this for all the other files in the future. 
fix commit - https://github.com/wso2/identity-apps/pull/4202/commits/2fb07a3b94df453ac29f221d3e86fd95a7296292